AuthenticationAndAuthorization/DataAccounts/Entitys/UserConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Security.Claims;
+
+namespace DataAccounts
+{
+    public static class UserConstants
+    {
+        public const string Admin = "Admin";
+        public const string User = "User";
+
+        public static IReadOnlyList<string> GetRole()
+        {
+            var userRoles = typeof(UserConstants)
+            .GetFields()
+            .Select(field => field.GetValue(null)?.ToString())
+            .ToList();
+
+            return userRoles.AsReadOnly()!;
+        }
```

Comment:
```suggestion
        public static readonly IReadOnlyList<string> Roles = new List<string> { Admin, User };
```

Reply:
because your constants might contain not roles only and your reflection implementation will cause bugs